scripts/test_mkn.py

```
@@ -0,0 +1,149 @@
+import subprocess
+import json
+import sys
+import re
+
+def run_cmd(args):
+    proc = subprocess.run(args, capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)